splinepy/helpme/create.py

```diff
@@ -277,6 +284,134 @@ def from_bounds(parametric_bounds, physical_bounds):
     return bspline_box
 
 
+def determinant_spline(self, dmin=False, show=False):
+    """
+    Creates the jacobian determinant spline of the given spline object.
+    Only works if (parameter dimension == physical dimension) and non-rational splines
+    or rational splines with equal weights
+    (otherwise give B-Spline approximation of spline)!
+
+    references:
+    Mantzaflaris, A., Jüttler, B., Khoromskij, B. N., & Langer, U. (2017)
+    Limkilde, A., Evgrafov, A., Gravesen, J., & Mantzaflaris, A. (2021)
+
+    Parameters
+    ----------
+    dmin: bool, optional
+      returns dmin of determinant spline
+      dmin > 0 --> spline is not tangled
+      dmin <= 0 --> spline is COULD BE tangled
```

Comment:
```suggestion
      dmin >= 0 --> spline is not tangled
      dmin < 0 --> spline is COULD BE tangled
```

Reply:
Please put this into the description of the method. We can help with the interpretation, but returning the spline to the user enough

Reply:
Okay!
And shouldn't "dmin <= 0" be correct according to literature for a possible tangled spline?

![image](https://github.com/tataratat/splinepy/assets/145120789/a9a65036-b1e3-469b-b0f6-ba5018ae5664)